supporting_content/columns/skupriceid.md

```diff
@@ -38,15 +38,15 @@ Current values observed in billing data for various scenarios:
 
 The following table serves as the basis for reviewing the SkuPriceId spec, as well as price, cost, quantity metrics, etc., impacted by the ChargeCategory and ChargeClass columns cleanup
```

Comment:
Introducing a minor irrelevant change to check if the issue with checks still remains.

```suggestion
The following table serves as the basis for reviewing the SkuPriceId spec, as well as price, cost, quantity metrics, etc., impacted by the ChargeCategory and ChargeClass columns cleanup.
```